app/views/sites/usage_rules/edit.html.slim

```diff
@@ -48,9 +48,6 @@
           = form.inputs "Service Plan Changing", class: "use-service-plans #{hide_class_unless_service_plans_visible(@settings)}" do
             = render 'shared/plan_change_settings', form: form, permission: :change_service_plan_permission, label: ''
 
-        - if can? :see, :end_users
-          = form.input :end_user_plans_ui_visible
```

Comment:
I guess we should also filter the param from here:
https://github.com/3scale/porta/blob/43383d8fe110b8ce25954a1748eb12b589966ace/app/controllers/sites/usage_rules_controller.rb#L10
And I dunno if there is an API for this or not 😄

Reply:
this part is only about UI.